source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md

```diff
@@ -0,0 +1,215 @@
+
+1. Review the sections below for important information about your site, including an explanation of the directory structure, Continuous Integration functions, and essential configuration actions.
+
+## Review Important Directories and Update File Paths
+
+### /web Directory
+
+Your site is stored and served from the `/web` subdirectory in the `pantheon.yml` file. You must store your website in this subdirectory for a Composer-based workflow. Placing your website in the subdirectory also allows you  to store tests, scripts, and other files related to your project in your repo without affecting your web document root. It also provides additional security by preventing web access through Pantheon. Your files may still be accessible from your version control project if it is public. See the [`pantheon.yml` documentation](/pantheon-yml#nested-docroot) for details.
+
+1. Verify that your website is stored in the `/web` subdirectory.
+
+### /web/wp Directory
+
+Your directories and files within the `/web` directory are stored in different locations compared to a default WordPress installation. [WordPress allows installing WordPress core in its own directory](https://wordpress.org/support/article/giving-wordpress-its-own-directory/), which is necessary when installing WordPress with Composer. The overall layout of directories in the repo is similar to [Bedrock](https://github.com/roots/bedrock).
+
+1. Move the `WP_SITEURL` to the `/web/wp` directory to allow WordPress core functions to work correctly. 
+
+1. Review the `/web/wp-config.php` file for key settings and move other files to the to the `/web/wp` directory as necessary.
+
+### composer.json File
+
+This project uses Composer to manage third-party PHP dependencies. Some files, such as `web/wp` may not be visible in the repository. This is because WordPress core and its plugins are installed via Composer and ignored in the `.gitignore` file.
+
+A custom, [Composer version of WordPress for Pantheon](https://github.com/pantheon-systems/wordpress-composer/) is used as the source for WordPress core.
+
+Third party WordPress dependencies, such as plugins and themes, are added to the project via `composer.json` file. The `composer.lock` file keeps track of the exact dependency version. Composer installer-paths are used to ensure the WordPress dependencies are downloaded into the appropriate directory.
+
+Non-WordPress dependencies are downloaded to the `/vendor` directory.
+
+1. Place all dependencies in the **require** section of your `composer.json` file. 
+
+    - This includes dependencies that are only used in non-Live environments. All dependencies in the **require** section are pushed to Pantheon.
+
+1. Place all dependencies that are not a part of the web application but are necessary to build or test the project in the **require-dev** section.
+
+    - Some example dependencies are `php_codesniffer` and `phpunit`. Dev dependencies are not deployed to Pantheon.
```

Comment:
Dev dependencies are deployed to dev and multidev environments, but are not deployed to the test and live environments.